Dockerfile

```diff
@@ -11,10 +11,10 @@ ENV JENKINS_FOLDER /usr/share/jenkins
 
 # Build Args
 ARG LIBMESOS_DOWNLOAD_URL=https://downloads.mesosphere.io/libmesos-bundle/libmesos-bundle-1.12.0.tar.gz
```

Comment:
bump libmesos to 1.4

Reply:
Good catch, following up with the team to see which version we can ship.

Reply:
let's use 1.14.0-beta for now.